lightningd/invoice.c

```diff
@@ -682,7 +682,7 @@ static struct command_result *json_invoice(struct command *cmd,
 	const u8 **fallback_scripts = NULL;
 	u64 *expiry;
 	struct sha256 rhash;
-	struct sha256 *description_hash;
```

Comment:
You might want to keep it as a pointer tu support both parameter and long descriptions